### PR TITLE
Write initial Rune interface

### PR DESCRIPTION
**Pull request by bricerising**
This includes a struct with some metadata about the rune and a run command that executes a docker command.

This thing is very ugly and happy path right now

### CODE REVIEW

**Comment by theinfamousrj** [comment]
I'm no Golang pro but it looks good to me.

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Gopkg.toml example
```
Comment (Dizdarevic) [inline Gopkg.toml]: Let's decide on if we'd rather use Deps or just use Go mods and have the mod file keep track of all the dependencies?

File: runes/rune_test.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	"testing"
+
+	"github.com/sygaldry/sygaldry-core/runes"
+)
+
+// TestRuneRun tests rune.Run
+func TestRuneRun(t *testing.T) {
+	myFirstRune := runes.NewRune("hello-world", []string{"TEST=test"})
+	if error := myFirstRune.Run(); error != nil {
+		t.Error("Expected success, got ", error)
```
Comment (Dizdarevic) [inline runes/rune_test.go]: I was getting “repository name must be canonical” when running it, so I replaced the image name in the run with “docker.io/library/node:7.8.0-alpine” and then got this error “Error response from daemon: page not found”. looks like it’s actually trying to pull a docker image when I run the tests

Comment (bricerising) [reply runes/rune_test.go]: Fixed!

**Pull request merged**